lib/games.js

```diff
@@ -1328,6 +1328,14 @@ export const games = {
     },
     release_year: 2015
   },
+  theisleevrima: {
```

Comment:
CI returns
```
IDFail {
    game_id: "theisleevrima",
    game_name: "The Isle Evrima",
    expected_id: "tie",
    rule_stack: [
        MoreThanTwoWordsMakeAcronym,
    ],
}
```
so the gid should be `tie`.

Reply:
Should I change just the gid or the protocol too? The file name from `theisleevrima.js` to `tie.js`

Reply:
Just the gid, the protocol could be explicit about its name (: